bin/shoryuken

```diff
@@ -21,7 +21,7 @@ module Shoryuken
       method_option :daemon,      aliases: '-d', type: :boolean, desc: 'Daemonize process'
       method_option :queues,      aliases: '-q', type: :array,   desc: 'Queues to process with optional weights'
       method_option :require,     aliases: '-r', type: :string,  desc: 'Dir or path of the workers'
-      method_option :config_file, aliases: '-C', type: :string,  desc: 'Path to config file'
+      method_option :config,      aliases: '-C', type: :string,  desc: 'Path to config file'
```

Comment:
I'd suggest supporting both (given that `config` was introduced in v3). Specifically, I suggest adding `method_option` for both parameters `config` and `config_file` which are redundant but raise an exception if *both* are specified. Then, in the `Shoryuken#start` method, I'd simply set `options[:config] = options[:config_file]` if it's specified.